android/assets/jsons/Civ V - Vanilla/Units.json

```diff
@@ -995,6 +995,22 @@
 			"-[33]% Strength vs [City]", "+[50]% Strength vs [wounded units]"],
 		"attackSound": "horse"
 	},
+	{
+		"name": "Hussar",
+		"unitType": "Mounted",
+		"replaces": "Cavalry",
+		"uniqueTo": "Austria",
+		"movement": 5,
+		"strength": 34,
+		"cost": 225,
+		"requiredTech": "Military Science",
+		"obsoleteTech": "Combustion",
+		"requiredResource": "Horses",
+		"upgradesTo": "Landship",
+		"uniques": ["Can move after attacking","No defensive terrain bonus","-[33]% Strength vs [City]" ,
+		"[+1] Visibility Range", "[+50]% Flank Attack modifier" ],
```

Comment:
I personally like the wording "[+50]% Strength when flanking and attacking" more, as it more closely follows the "[amount]% Strength <conditions>" template most uniques use

Reply:
That's not what the bonus is. It's a bonus to the flanking modifier itself, ie instead of +10% per adjacent friendly unit you get +15%. Perhaps there is a clearer way to express that?

Reply:
Knowing what it is it is quite a clear way to describe it. Only things I could come up with was "Bonus from flanking [+50]%", or maybe "Strength bonus from flanking [+50]%", but if you don't like those and don't have a better idea feel free to leave it as is